src/service-override/files.ts

```diff
@@ -469,7 +469,7 @@ const userDataFileSystemProvider = new InMemoryFileSystemProvider()
 void userDataFileSystemProvider.mkdir(URI.from({ scheme: Schemas.vscodeUserData, path: '/User/' }))
 
 const providers: Record<string, IFileSystemProvider> = {
-  extension: extensionFileSystemProvider,
+  'extension-fs': extensionFileSystemProvider,
```

Comment:
it probably needs to be changed in extensions.ts in `registerExtension` as well

Reply:
I checked that, but I _think_ that's not needed as it uses the filesystem directly; https://github.com/CompuIves/monaco-vscode-api/blob/25fbe41459c2fc3cdb4a1594060b6cecb5cce51e/src/service-override/files.ts#L513-L515

Reply:
I mean here: https://github.com/CompuIves/monaco-vscode-api/blob/25fbe41459c2fc3cdb4a1594060b6cecb5cce51e/src/extensions.ts#L100

Reply:
Oh good shout! Fixed it.

Reply:
probably nitpicking, but maybe `extension-fs` is not a so good descriptive name, since `extension` scheme will also be a fs to store extensions.

Also, I'm not sure to understand how it's supposed to work, what will use the `extension` fs provider? I don't see any registration of it, neither in that library or in VSCode itself

Reply:
I've just tried keeping `extension` scheme and everything seems to work fine, what is the issue?